### DISCUSSION TITLE
FEATURE REQUEST:  Detail Row Expand, allow one at a time

### DISCUSSION THREAD

**Discussion by masterbater** [KevinVandy/mantine-react-table#51] [Ideas]
Hello is it possible to add a props for making it that detail row expand will be one at a time. Means if there is an expanded row then click on a new row, it will close the previous one then open the new one. 
I dont know if this is already built in or can be achieved by onExpandedChange, but I dont see it in the documentation. 
Thanks for making this project love your material-react-table

**Comment by KevinVandy** [comment]
You could set the expanded state manually in the onClick of the expand buttons to just be the 1 row id
